src/utils/reports.ts

```diff
@@ -1,125 +1,11 @@
-import { difference, get, omit, memoize } from 'lodash';
-import { Network } from '../types';
-import { querySubgraphData } from './apisRequest';
+import { difference, omit, memoize } from 'lodash';
+import { Network, StratReportGraphType } from '../types';
+import { getService } from '../services/VaultService';
 
 import { StrategyReportContextValue } from '../contexts/StrategyReportContext';
 
 const OMIT_FIELDS = ['results', 'transaction', 'id'];
 
-const buildReportsQuery = (strategies: string[]): string => `
```

Comment:
Thie util file contained all the subgraph Strategy Report code. Its been pulled out & refactored into the Subgraph class